### PR TITLE
ENYO-2486 ENYO-3012 Add edge detection, fix docs

### PR DESCRIPTION
**Pull request by webOS101**
Enyo-DCO-1.1-Signed-off-by: Roy Sutton <roy.sutton@lge.com>

### CODE REVIEW

File: src/platform.js [hunk 1]
```diff
@@ -35,17 +36,28 @@ var utils = require('./utils');
...
+	* `true` if the platform has native double-finger [events]{@glossary event}.
+	* @public
+	*/
 	gesture: Boolean(('ongesturestart' in window) || window.navigator.msMaxTouchPoints)
+
+	/**
+	* The name of the platform that was detected or `undefined` if the platform
+	* was unrecognized. This value is the key name for the major version of the
+	* platform on the exported object.
+	* @var {string} platformName
```
Comment (aarontam) [inline src/platform.js]: What does `@var` mean in JSDoc?

Comment (webOS101) [inline src/platform.js]: http://usejsdoc.org/tags-member.html  We're documenting a virtual member.  I suppose that @member will work as well... But I thought I tried it and settled on @var.  I can do a quick test again with @member to be sure.

Comment (aarontam) [inline src/platform.js]: Gotcha - for some reason that synonym didn't come up in the search. I'm not opposed to it, but we seem to be using `@member` everywhere else.

File: src/platform.js [hunk 2]
```diff
@@ -35,17 +36,28 @@ var utils = require('./utils');
...
+	* `true` if the platform has native double-finger [events]{@glossary event}.
+	* @public
+	*/
 	gesture: Boolean(('ongesturestart' in window) || window.navigator.msMaxTouchPoints)
+
+	/**
+	* The name of the platform that was detected or `undefined` if the platform
+	* was unrecognized. This value is the key name for the major version of the
+	* platform on the exported object.
+	* @member {string} platformName
```
Comment (aarontam) [inline src/platform.js]: Oh missed this earlier, shouldn't this be `{String}`?

**Pull request merged**